docs/configuring.md

```diff
@@ -63,6 +79,15 @@ codemods:
     - pixee:python/use-walrus-if
 ```
 
+#### `catalog`
+A string value that determines the catalog level for code improvements. The catalog value can be set to `less`, `moderate`, or `more`.
```

Comment:
A string value that determines the codemod catalog used for code improvements. The value can be set to...